### DISCUSSION TITLE
`fusermount` vs `fusermount3`

### DISCUSSION THREAD

**Discussion by infeo** [libfuse/libfuse#744] [Q&A]
On many linux systems, libfuse2 and libfuse3 are installed side-by-side. Hence, on the system `fusermount` from libfuse2  as well as `fusemount3` from the 3.x branch is present.

My questions:
* Can I safely unmount a libfuse3-fs mount with `fusermount`?
* Can I safely unmount a libfuse2-fs mount with `fusermount3`?
* Are there incompabilities between `fusermount` and `fusermount3`?

Thank you in advance.

**Comment by Nikratio** [comment]
They are incompatible in general, so I would never consider this safe.

However, in many cases fusermount3 is able to unmount a fuse2 filesystem.
